tests/wamr-test-suites/spec-test-script/all.py

```diff
@@ -91,7 +91,7 @@ def ignore_the_case(
         return True
 
     if gc_flag:
-        if case_name in ["type-canon", "type-equivalence", "type-rec"]:
+        if case_name in ["type-canon", "type-equivalence", "type-rec", "array_init_elem", "array_init_data"]:
```

Comment:
The cases in type-canon.wast are simple, maybe we can remove "type-canon" here?
https://github.com/WebAssembly/gc/blob/master/test/core/type-canon.wast

Reply:
Done, thanks!